webpack.config.js

```diff
@@ -24,166 +23,81 @@ const { dependencies } = require( './package' );
 
 const WORDPRESS_NAMESPACE = '@wordpress/';
 
-/**
- * Given a string, returns a new string with dash separators converted to
- * camelCase equivalent. This is not as aggressive as `_.camelCase` in
- * converting to uppercase, where Lodash will also capitalize letters
- * following numbers.
- *
- * @param {string} string Input dash-delimited string.
- *
- * @return {string} Camel-cased string.
- */
-function camelCaseDash( string ) {
-	return string.replace(
-		/-([a-z])/g,
-		( match, letter ) => letter.toUpperCase()
-	);
-}
-
 const gutenbergPackages = Object.keys( dependencies )
 	.filter( ( packageName ) => packageName.startsWith( WORDPRESS_NAMESPACE ) )
 	.map( ( packageName ) => packageName.replace( WORDPRESS_NAMESPACE, '' ) );
 
-const externals = {
-	react: 'React',
-	'react-dom': 'ReactDOM',
-	tinymce: 'tinymce',
-	moment: 'moment',
-	jquery: 'jQuery',
-	lodash: 'lodash',
-	'lodash-es': 'lodash',
+config.entry = gutenbergPackages.reduce( ( memo, packageName ) => {
+	const name = camelCaseDash( packageName );
+	memo[ name ] = `./packages/${ packageName }`;
+	return memo;
+}, {} );
+
+config.output = {
+	filename: './build/[basename]/index.js',
+	path: __dirname,
+	library: [ 'wp', '[name]' ],
+	libraryTarget: 'this',
 };
 
-gutenbergPackages.forEach( ( name ) => {
-	externals[ WORDPRESS_NAMESPACE + name ] = {
-		this: [ 'wp', camelCaseDash( name ) ],
-	};
-} );
-
-const isProduction = process.env.NODE_ENV === 'production';
-const mode = isProduction ? 'production' : 'development';
-
-const config = {
-	mode,
-	entry: gutenbergPackages.reduce( ( memo, packageName ) => {
-		const name = camelCaseDash( packageName );
-		memo[ name ] = `./packages/${ packageName }`;
-		return memo;
-	}, {} ),
-	output: {
-		filename: './build/[basename]/index.js',
-		path: __dirname,
-		library: [ 'wp', '[name]' ],
-		libraryTarget: 'this',
-	},
-	externals,
-	resolve: {
-		modules: [
-			__dirname,
-			'node_modules',
-		],
-		alias: {
-			'lodash-es': 'lodash',
+config.plugins.push(
+	// Create RTL files with a -rtl suffix
+	new WebpackRTLPlugin( {
+		suffix: '-rtl',
+		minify: config.mode === 'production' ? { safe: true } : false,
+	} ),
+	new CustomTemplatedPathPlugin( {
+		basename( path, data ) {
+			let rawRequest;
+
+			const entryModule = get( data, [ 'chunk', 'entryModule' ], {} );
+			switch ( entryModule.type ) {
+				case 'javascript/auto':
+					rawRequest = entryModule.rawRequest;
+					break;
+
+				case 'javascript/esm':
+					rawRequest = entryModule.rootModule.rawRequest;
+					break;
+			}
+
+			if ( rawRequest ) {
+				return basename( rawRequest );
+			}
+
+			return path;
 		},
-	},
-	module: {
-		rules: [
-			{
-				test: /\.js$/,
-				use: [ 'source-map-loader' ],
-				enforce: 'pre',
-			},
-			{
-				test: /\.js$/,
-				exclude: [
-					/block-serialization-spec-parser/,
-					/is-shallow-equal/,
-					/node_modules/,
-				],
-				use: 'babel-loader',
-			},
-		],
-	},
-	plugins: [
-		new DefinePlugin( {
```

Comment:
We seem to have dropped the version flags config in an update/rebase.

Reply:
Yeah, great catch :eagle: :eye:

Reply:
Fixed in https://github.com/WordPress/gutenberg/pull/13814/commits/c1d98342123d0b1430b4c2f4b214bd9cb6df888f.